lib/ui/hooks.dart

```diff
@@ -253,6 +254,8 @@ void _invoke(void callback()?, Zone zone) {
 }
 
 /// Invokes [callback] inside the given [zone] passing it [arg].
+///
+/// The 1 in the name refers to the number of arguments expected by the callback.
```

Comment:
...as well as the number of arguments passed to this function, not counting `callback` and `zone`.

Reply:
added